index.bs

```diff
@@ -764,20 +770,13 @@ WorkerNavigator includes NavigatorML;
 
 ## {{ML}} interface ## {#api-ml}
 <script type=idl>
-enum MLDeviceType {
-  "cpu",
-  "gpu",
-  "npu"
-};
```

Comment:
Hmm, I wonder why "Changes from all commits" is showing this as a deletion, assuming the merge from main went cleanly?

![image](https://github.com/user-attachments/assets/64f8f9a3-7511-42d8-96aa-dfe2758a6589)

Guessing GitHub glitch.

Reply:
Hmmm, my local history seems a bit off... investigating...

Reply:
Well, it will all be squashed in the merge anyway.

Reply:
Weird. Not sure what happened - the Zoltan's change was in the branch history twice. I ended up rebasing/force-pushing... looks okay now?